### PR TITLE
CI: stop running coveralls in forks

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Not sure if this will work for PRs... It probably won't. My plan is to stop this from running when we are not on our repo or a PR against our repo.

File: .github/workflows/js.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ jobs:
 
       - name: Run Coveralls
         uses: coverallsapp/github-action@v2
+        if: github.repository == 'twbs/bootstrap'
```
Comment (julien-deramond) [inline .github/workflows/js.yml:49]: Maybe both are working but I've seen and used another approach to do that:
```suggestion
        if: github.repository_owner == 'twbs'
```

Comment (XhmikosR) [reply .github/workflows/js.yml:49]: I think we could probably just use this too:

```
if: ${{ !github.event.repository.fork }}
```

**Review by louismaximepiton — APPROVED** [review]
Just tried it on a forked Bootstrap and it seems to work quite well. Didn't tried inside a PR against our repo.

**Pull request merged**